docs/src/main/paradox/scala/http/routing-dsl/rejections.md

```diff
@@ -33,7 +33,7 @@ the @ref[handleRejections](directives/execution-directives/handleRejections.md#h
 ## The RejectionHandler
 
 The @ref[handleRejections](directives/execution-directives/handleRejections.md#handlerejections) directive delegates the actual job of converting a list of rejections to its argument, a
-@github[RejectionHandler](/akka-http/src/main/scala/akka/http/scaladsl/server/RejectionHandler.scala), which is defined like this:
```

Comment:
Linking to javadocs/scaladocs is more friendly than dropping readers into the source code and also allows us to link to a specific method instead of a line number which may change.